MANIFEST.in

```diff
@@ -2,10 +2,10 @@ include ChangeLog
 include CONTRIBUTORS.txt
 include COPYING
 include README.rst
-include bin/*
-include examples/*.py examples/pylintrc examples/pylintrc_camelcase
 include elisp/startup elisp/*.el
 include pytest.ini
-recursive-include doc *.bat *.png *.py *.svg *.rst *.txt Makefile
+recursive-include doc *.bat *.png *.py *.svg *.rst *.txt Makefile .md
 graft tests
```

Comment:
You probably want to remove this line here: `graft tests`. It will include all files in the `tests` folder recursively.

Reply:
Yeah, I was wondering if we keep the doc folder and the test folder in the packaged result at all... (We exclude .pyc file from tests directory on the next line).

Reply:
I would probably remove most lines in the `MANIFEST.in`. It's only relevant for the `sdist` to begin with and there isn't a good reason to keep them around if they aren't used.

```
include ChangeLog             # not used
include CONTRIBUTORS.txt      # included through `license_files`
include COPYING               # included through `license_files`
include README.rst            # included `EGG-INFO` / `PKG-INFO`
include elisp/startup elisp/*.el  # ?
include pytest.ini            # not necessary
recursive-include doc *.bat *.png *.py *.svg *.rst *.txt Makefile .md  # Unused
recursive-exclude tests *.pyc .benchmark  # Can be removed if `[options.packages.find]` is added
recursive-exclude script *    # Same here
recursive-exclude examples *  # and here
```

Reply:
Yeap, I was waiting for sandrotosi's opinion for that :)